### DISCUSSION TITLE
Enchantment Combination

### DISCUSSION THREAD

**Discussion by roy0510** [Zrips/CMI#5195] [Q&A]
Hello good,

Ask:
It is possible to do a combination of enchantment on the anvil (/cmi anvil or anvil block), that is to say a book sharpness 6+6 or 7+7 subsequently, or a combination of a sword with sharpness 6,
I have tried turning `Enchanting:` `true` and `false`
and using the cmi.anvil.nolimits permission (I'm not sure what this permission does).

I am using the command **/cmi give enchantedbook 1 e sharpness:6** to give me the enchantment of the book.

**Comment by CloudeLecaw** [comment]
no its not possible the anvil enchantment is set in vanilla to level 2,3 or 5

you can do this but the permissions effect the enchat comment itself not the anviil ;)

with cmi.anvil.nolimits permission you can have the vanilla enchantsmant  in vanil if u reduced them for players
